django_mysql/models/functions.py

```diff
@@ -260,6 +260,17 @@ def __init__(self, expression, path=None, **extra):
         super(JSONLength, self).__init__(*exprs, output_field=output_field)
 
 
+class JSONValue(Func):
+    function = 'CAST'
+    template = '%(function)s(%(expressions)s as json)'
+    arity = 1
```

Comment:
no need for this as we are defining custom `__init__` so it's impossible to init with wrong arity